src/index.coffee

```diff
@@ -3,8 +3,8 @@ Path  = require('path')
 Hubot = require('hubot')
 
 class MockRobot extends Hubot.Robot
-  constructor: ->
-    super null, null, false, 'hubot'
+  constructor: (httpd=false) ->
```

Comment:
httpd (expressjs server) will not be closed on each test, so we should close server on `afterEach`.

```coffee
class Room extends Hubot.Adapter
destroy: ->
  @robot.server.close()
```

```coffee
afterEach ->
  @room.destroy()
```